packages/lightning-lsp-common/src/__tests__/workspace-type.test.ts

```diff
@@ -30,6 +30,18 @@ describe('detectWorkspaceType', () => {
         expect(workspaceType).toEqual(WorkspaceType.SFDX);
     });
 
+    test('when an lwc.config.js file is present, workspaceType is STANDARD_LWC', () => {
+        mockFs({
+            workspacedir: {
+                'lwc.config.js': '',
```

Comment:
@xyc This is not correct, as it's `lwc.config.json`, see [here](https://github.com/forcedotcom/lightning-language-server/pull/16/files#diff-fbeef00ae19df8f84ca0d4a7c8cee648R33). See also [here](https://rfcs.lwc.dev/rfcs/lwc/0020-module-resolution#lwc-configuration) in the docs.

Reply:
Nice catch. 
Here's a fix for this:
https://github.com/forcedotcom/lightning-language-server/pull/152/files

Thank you!